html-forms/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <h1>Coding Form</h1>
+  <label for="submitted-email">
+    <div>Email</div>
+    <input id="submitted-email">
+  </label>
+
+  <p>Why is coding cool?</p>
+   <label for ="why-cool">
+    <select name="dropdown" id="why-cool">
+      <option value="option">Select an Option</option>
+      <option value="option">Because it's interesting</option>
+      <option value="option">Because it's fun</option>
+      <option value="option">Because it's powerful</option>
+    </select>
+  </label>
+  <br>
+
+  <label for="language"></label>
+    <form id ="language" action="www.example.com">
```

Comment:
You should not have individual `form` elements for each input, every input in a form should be surrounded by one `form` element. Please remove the extras here, and add one around your entire form.